src/__tests__/radium-test.js

```diff
@@ -19,6 +19,20 @@ describe('Radium blackbox tests', () => {
     sandbox.restore();
   });
 
+  it('returns only one display attribute', () => {
+    @Radium class TestComponent extends Component {
+      render() {
+        return <div style={{display: 'flex'}} />;
+      }
+    }
+
+    const output = getRenderOutput(<TestComponent />);
+
+    expect(output.props.style).to.deep.equal({
+      display: 'flex'
```

Comment:
This returns an array of `['-webkit-box', 'moz-box', '-ms-flexbox', 'webkit-flex', 'flex']`.

Reply:
Perhaps this test should try to match `{ display: "-webkit-box;display:moz-box;display:-ms-flexbox;display:webkit-flex;display:flex" }` instead.